overlays/nodejs/serverless/function.ts

```diff
@@ -155,12 +159,13 @@ export class Function extends pulumi.ComponentResource {
         });
 
         let codePaths = computeCodePaths(closure, serializedFileName, options.includePaths, options.includePackages);
-        
+
         // Create the Lambda Function.
         this.lambda = new lambda.Function(name, {
             code: new pulumi.asset.AssetArchive(codePaths),
             handler: serializedFileName + "." + handlerName,
             runtime: options.runtime || lambda.NodeJS8d10Runtime,
+            environment: options.environment,
```

Comment:
Note that you'll either need to make the `options.environment` use the nested `variables` property as on the underlying `aws.lambda.FunctionArgs`, or you will need to pass the `options.environment` as the `variables` property of this object.

Probably worth aligning with `aws.lambda.FunctionArgs`, which will require changing the signature above.

Reply:
I changed the signature above, so this should no longer apply.